bin/duh-export-scala.js

```diff
@@ -28,11 +28,27 @@ const argv = yargs
   .help()
   .argv;
 
+const fixupPortDirections = duh => new Promise((resolve) => {
+
+  // expand ports
+  if (duh.component !== undefined) {
+    const model = duh.component.model;
+    model.ports = model.ports.map(port => {
+      const wire = port.wire;
+      wire.direction = wire.analog ? wire.analog : wire.direction;
+      return port;
```

Comment:
Should this be happening here at this layer of the tool? Will the core part of the code (under `lib/*`) ever need to differentiate between the Verilog direction and the actual analog direction? Also, will the blackboxing work properly if the analog direction is different than the actual Verilog module's direction?

Reply:
I think this is the right place to put this. This is kinda a hack, because all the export scala code expects to get the "actual" direction from the `direction` field so this fixup saves us from having add `analog` checks everywhere. duh-core doesn't really have to do any interpretation of what the values of the fields mean or anything

Reply:
and it does seem like blackboxing with different directions works